packages/backend-core/src/cache/docWritethrough.ts

```diff
@@ -0,0 +1,85 @@
+import { AnyDocument, Database } from "@budibase/types"
+
+import { JobQueue, createQueue } from "../queue"
+import * as dbUtils from "../db"
+import { logWarn } from "../logging"
+
+interface ProcessDocMessage {
+  dbName: string
+  docId: string
+  data: Record<string, any>
+}
+
+export const docWritethroughProcessorQueue = createQueue<ProcessDocMessage>(
+  JobQueue.DOC_WRITETHROUGH_QUEUE,
+  {
+    jobOptions: {
+      // We might have plenty of 409, we want to allow running almost infinitely
+      attempts: Number.MAX_SAFE_INTEGER,
+    },
+  }
+)
+
+class DocWritethroughProcessor {
+  init() {
+    docWritethroughProcessorQueue.process(async message => {
+      try {
+        await this.persistToDb(message.data)
+      } catch (err: any) {
+        if (err.status === 409) {
+          logWarn(`409 conflict in doc-writethrough cache`)
```

Comment:
We've had issues with stuff like this in the past spamming - this is a valid situation, I think we would only want to log warn if it happens too many times in a row, e.g. if its on its max attempt or something.

Reply:
Removed and logging the attempt count on the error